pkg/apis/app/v2/kieapp_types.go

```diff
@@ -252,6 +252,8 @@ type KieAppObject struct {
 	ImageTag string `json:"imageTag,omitempty"`
 	// The storageClassName to use
 	StorageClassName string `json:"storageClassName,omitempty"`
+	// Enable the clustering between Kie Servers and add one replicas if only one is configured
+	JbpmCluster *bool `json:"jbpmCluster,omitempty"`
```

Comment:
if this is only used by kie servers, this field should be moved to `KieServerSet` object instead

Reply:
ok